packages/lib/src/vis/line/glyphsGeometry.ts

```diff
@@ -1,44 +1,40 @@
-/* eslint-disable @typescript-eslint/no-non-null-assertion */
-import { type IgnoreValue, type NumArray } from '@h5web/shared/vis-models';
+import { type BufferAttribute, BufferGeometry } from 'three';
 
-import { type AxisScale } from '../models';
-import H5WebGeometry from '../shared/h5webGeometry';
+import { type H5WebGeometry } from '../models';
 import { createBufferAttr, Z_OUT } from '../utils';
+import { type LineGeometryParams } from './lineGeometry';
 
-interface Params {
-  abscissas: NumArray;
-  ordinates: NumArray;
-  abscissaScale: AxisScale;
-  ordinateScale: AxisScale;
-  ignoreValue?: IgnoreValue;
-}
```

Comment:
I removed a bit of duplication by exporting `LineGeometryParams` from the line geometry.